src/components/pages/profile/[name]/registration/steps/Complete.tsx

```diff
@@ -193,6 +199,22 @@ const Complete = ({
   const { width, height } = useWindowSize()
   const { InvoiceFilled, avatarSrc } = useEthInvoice(name, isMoonpayFlow)
 
+  const nameWithColourEmojis = useMemo(() => {
+    const data = tokenise(beautifiedName)
+    return data.map((item, i) => {
+      if (item.type === 'emoji') {
+        const string = String.fromCodePoint(...item.emoji)
```

Comment:
Nit pick. Not ideal to use "string" as the variable name.

Reply:
true, fixed